src/components/SideNavBar/OldSideNavBar.jsx

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import style from './oldStyle.scss';
+
+class SideNavBar extends Component {
```

Comment:
should this be a stateless component then? I think those are supposed to be more performant eventually (in near future versions of React)